client/src/api/editApplicantProfile.js

```diff
@@ -0,0 +1,14 @@
+export const getSkillsForEditApplicantProfile = id => {
```

Comment:
There is no point for this function already you have a function to get an applicant skills

Reply:
yea I have the function that gets the skills as a name but I need the id here